rn/Teacher/src/routing/register-screens.js

```diff
@@ -65,7 +65,6 @@ import FilesList from '../modules/files/FilesList'
 import EditFile from '../modules/files/EditFile'
 import EditFolder from '../modules/files/EditFolder'
 import ViewFile from '../modules/files/ViewFile'
-import PagesList from '../modules/pages/list/PagesList'
```

Comment:
@GabrielPalmer This is what is causing your lint error. You need to bring this back since it is used for Teacher.